media-plugins/vdr-devstatus/vdr-devstatus-0.4.1-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR plugin: display the usage status of the available DVB devices"
+HOMEPAGE="http://www.u32.de/vdr.html"
+SRC_URI="http://www.u32.de/download/${P}.tgz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+
+DEPEND=">=media-video/vdr-1.6.0"
+RDEPEND="${DEPEND}"
+
+QA_FLAGS_IGNORED="
+	usr/lib/vdr/plugins/libvdr-.*
+	usr/lib64/vdr/plugins/libvdr-.*"
+
+src_prepare() {
+	# remove untranslated .po files
+	rm "${S}"/po/{ca_ES,cs_CZ,da_DK,el_GR,et_EE,hr_HR,hu_HU,nl_NL,nn_NO,pl_PL,pt_PT,ro_RO,sl_SI,sv_SE,tr_TR}.po \
+		|| die "failed to remove untranslated .po files"
+
+	vdr-plugin-2_src_prepare
+
+	# some patches from debian/e-tobi
+	eapply "${FILESDIR}/recordingfound.patch"
```

Comment:
I see you added these patches. Remember the naming "rules" we try to follow, 
https://dev.gentoo.org/~vapier/clean-patches
Ie add `${P}` to patch name.

Reply:
Have read that page and renamed patches accordingly.